.changeset/config.json

```
@@ -0,0 +1,20 @@
+{
+  "changelog": ["@changesets/changelog-github", { "repo": "WalletConnect/walletconnect-monorepo" }],
+  "commit": true,
```

```suggestion
  "commit": false,
```
`v2.0` is protecrted and this will fail